environment-creator-application/src/main/java/com/ca/apim/gateway/cagatewayconfig/EnvironmentCreatorApplication.java

```diff
@@ -88,7 +88,9 @@ void run() {
                 templatizedBundleFolderPath,
                 environmentConfigurationFolderPath,
                 APPLICATION,
-                "_0_env.req.bundle"
+                "_0_env.req.install.bundle",
```

Comment:
Is `gw7` build running successfully? Also, all JUnits are working after this change?

Reply:
yes